doc/languages-frameworks/elisp.section.md

```diff
@@ -0,0 +1,75 @@
+# Emacs Lisp {#sec-emacs-lisp}
+
+Since at least version 24, Emacs has a native package manager, `package.el`. Since its inception, many Elisp packages were created by Emacs community at large, as well as package repositories.
+
+<!--
+TODO: add a chapter about melpaBuild and the general Elisp Nix framework
+-->
+
+## Bulk Update {#sec-emacs-lisp-packages-bulk-update}
+
+The chief Elisp package repositories are:
+
+- [GNU ELPA](https://elpa.gnu.org/),
+- [NonGNU ELPA](https://elpa.nongnu.org/), and
+- [MELPA](https://melpa.org/).
+
+Nixpkgs provides a comprehensive infrastucture that leverages the contents of these Elisp repositories as Nix packages. Here we will the bulk-updating automation tooling.
+
+Inside the `pkgs/applications/editors/emacs/elisp-packages` subdirectory we find the following scripts:
+
+- `update-scripts-library.sh`
```

Comment:
Should we add "overlay" to its name since it is tightly coupled with `emacs-overlay`?

- `download_packagesets` is tightly coupled with `emacs-overlay`
- What is the use case for `commit_packagesets` without involving `emacs-overlay`?

Reply:
> What is the use case for commit_packagesets without involving emacs-overlay?

Whether the packageset is obtained by downloading from emacs-overlay or by local generation, committing it to Nixpkgs follows the same steps: commit the file only when it is modified.